.mailmap

```diff
@@ -0,0 +1,61 @@
+Aaron D. Campbell <aaroncampbell@git.wordpress.org> <aaroncampbell@602fd350-edb4-49c9-b593-d223f7449a82>
+Aaron Jorbin <jorbin@git.wordpress.org> <jorbin@602fd350-edb4-49c9-b593-d223f7449a82>
+Adam Silverstein <adamsilverstein@git.wordpress.org> <adamsilverstein@602fd350-edb4-49c9-b593-d223f7449a82>
+Andrea Fercia <afercia@git.wordpress.org> <afercia@602fd350-edb4-49c9-b593-d223f7449a82>
+Andrew Fercia <afercia@git.wordpress.org> <afrecia@602fd350-edb4-49c9-b593-d223f7449a82>
```

Comment:
```suggestion
Andrea Fercia <afercia@git.wordpress.org> <afercia@602fd350-edb4-49c9-b593-d223f7449a82>
Andrew Fercia <afercia@git.wordpress.org> <afrecia@602fd350-edb4-49c9-b593-d223f7449a82>
```

Is this a dupe, I'm not seeing `Andrew Fercia` on trunk (which is nice, because it';s not Andrea's name).

Is there a chance it was intended to be Andrew Duthie in amongst all the Andrews?

Reply:
oh no this was likely a mistake on my part. I started manually creating the list from `git shortlot -n -s -e` but then realized that all these GUID-based emails were auto-generated and moved to a script. Probably I accidentally typed "Andrew" when it should have been "Andrew" and failed to notice when I merged the lists.

Reply:
Removed the incorrect line in 379b0f4f87